client/src/Components/UserSignUp.js

```diff
@@ -34,13 +35,61 @@ function SignUp() {
   }));
   const classes = useStyles()
 
+  const SignUp = props => {
+    // Declare and initialize state variables
+    let [firstname, setFirstname] = useState('')
+    let [lastname, setLastname] = useState('')
+    let [email, setEmail] = useState('')
+    let [message, setMessage] = useState('')
+    let [password, setPassword] = useState('')
+  
+  
+      useEffect(()=>{
+        setMessage('')   
+      }, [firstname, lastname, email, password])
+  
+      
+
+    const handleSubmit = e => {
+      e.preventDefault()
+      fetch(`${process.env.REACT_APP_SERVER_URL}/server/auth`,{
+        method: 'POST',
+        body: JSON.stringify({
+          firstname,
+          lastname,
+          email,
+          password
+        }),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
+       .then(response => {
+         if (!response.ok){
+           setMessage("Thank you for joining")
+         }
+  
+         response.json().then(result => {
+          props.createUser(result.token);
+        })
+       })
+      .catch(err => {
+        consile.log(err);
+        setMessage(`${err.toString()}`);
+      })
+  
+    }
+    if (props.user) {
+      return <Redirect to="../Components/Home" />
```

Comment:
This should redirect to just `/home` so that it triggers one of the defined routes in the `Content` component. That will take of actually bringing up the correct component.